modules/custom-post-types/js/many-items.js

```diff
@@ -1,27 +1,27 @@
 /* jshint onevar: false, smarttabs: true */
```

Comment:
Noting that I don't see these options in https://jshint.com/docs/options/

Otherwise, they seemed like something that would've conflicted with Prettier.

Reply:
Those options[ were removed](https://github.com/jshint/jshint/releases/tag/2.5.0) from `jshint` 2.5.0 four years ago! :laughing: 

JP is on `jshint` 2.9.7 now, probably fine to remove already.

Reply:
```suggestion
```

Reply:
Doesn't need to be in this PR, seems tedious task as there might be valid options mixed in between.

Seems like easier to remove them all at once together with jshint.